draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
```

Comment:
I think EKR would like some general theory of operation note that gets people the big idea of what algo is going to do. I am thinking something along lines of 

The main idea is when a RID is present, it overrides any SSRC information in deciding how to route the packet but updates the SSRC map such that future packets on the same SSRC but no RID get routed the same way. If no RIDs are ever received, and SSRC signaling information is known, then that can be used but RID information overrides SSRC if they happen to conflict as some architectures  may have devices that change the SSRC but nothing should be changing the RIDs.